src/code_800431B0.c

```diff
@@ -144,469 +144,59 @@ s32 func_80043A54(s32 arg0) {
     return phi_v1;
 }
 
-#ifdef MIPS_TO_C
-//generated by mips_to_c commit 3c3b0cede1a99430bfd3edf8d385802b94f91307
-
-
-void func_80043A84(s32 arg0, s32 arg1, s32 arg2) {
```

Comment:
iirc (it's quite possibly I don't)

I may have very well been messing with this function (or a similar one) trying to figure out how to add those if statements into it... And it's just loadtexblock... lol